### PR TITLE
aug sl release notes

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds the release notes for the SL for the month of aug.

[ ] Need PM review

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes.md [hunk 1]
```diff
@@ -19,7 +19,11 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 \* The official release date for this new format of release notes is May 15th, 2024. Historical release notes for prior dates may not reflect all available features released earlier this year or their tenancy availability.
 
 ## August 2024
-- **New**: You can now configure metrics at granularities at finer time grains, such as hour, minute, or even by the second. This is particularly useful for more detailed analysis and for datasets where high-resolution time data is required, such as minute-by-minute event tracking. Refer to [dimensions](/docs/build/dimensions) for more information about time granularity.
+- **New**: Configure metrics at finer time grains, such as hour, minute, or even by the second. This is particularly useful for more detailed analysis and for datasets where high-resolution time data is required, such as minute-by-minute event tracking. Refer to [dimensions](/docs/build/dimensions) for more information about time granularity.
```
Comment (nataliefiann) [inline website/docs/docs/dbt-versions/release-notes.md]:
```suggestion
- **New**: Configure metrics at finer time grains, such as an hour, minute, or even by the second. This is particularly useful for more detailed analysis and for datasets where high-resolution time data is required, such as minute-by-minute event tracking. Refer to [dimensions](/docs/build/dimensions) for more information about time granularity.
```

File: website/docs/docs/dbt-versions/release-notes.md [hunk 2]
```diff
@@ -19,7 +19,11 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
...
+- **Enhancement**: Microsoft Excel now supports saved selections and saved queries. Use Saved selections to save your query selections within the Excel application. The application also clears stale data in [trailing rows](/docs/cloud-integrations/semantic-layer/excel#other-settings) by default. To return your results and keep any previously selected data intact, un-select the **Clear trailing rows** option.
```
Comment (nataliefiann) [inline website/docs/docs/dbt-versions/release-notes.md]:
```suggestion
- **Enhancement**: Microsoft Excel now supports [saved selections](/docs/cloud-integrations/semantic-layer/excel#using-saved-selections) and [saved queries](/docs/cloud-integrations/semantic-layer/excel#using-saved-queries). Use Saved selections to save your query selections within the Excel application. The application also clears stale data in [trailing rows](/docs/cloud-integrations/semantic-layer/excel#other-settings) by default. To return your results and keep any previously selected data intact, un-select the **Clear trailing rows** option. 
```

Comment (nataliefiann) [reply website/docs/docs/dbt-versions/release-notes.md]: Hiya @mirnawong1 

This is a non-blocker suggestion.

Kind Regards
Natalie

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

Thank you for creating this PR - it's great! I've made one grammar suggestion and two non-blocker suggestions and approved this for you.

Kind Regards
Natalie

File: website/docs/docs/dbt-versions/release-notes.md (hunk 2 above)
Comment (mirnawong1) [reply website/docs/docs/dbt-versions/release-notes.md]: i dont think i should link to ghseets if we're talking about excel but I'll link to the excel one!

Comment (mirnawong1) [reply website/docs/docs/dbt-versions/release-notes.md]: will edit your suggestion  Natalie!

**Comment by mirnawong1** [comment]
approved by PM

**Pull request merged**